### ISSUE TITLE
PyTorch model

### ISSUE THREAD

**Issue by jinglescode** [yaoli90/Conv-CA#3]
Hi @yaoli90, could you implement PyTorch version for the correlation analysis layer `Corr`?

**Comment by dahao-1998** [comment]
+1

**Comment by yaoli90** [comment]
> +1

Work on it. Will be uploaded very soon.

**Comment by yaoli90** [comment]
Please check the pytorch re-implementation.

**Comment by dahao-1998** [comment]
Thank you very much。


2595386468
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yaoli90/Conv-CA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月27日(星期一) 晚上10:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yaoli90/Conv-CA] PyTorch model (Issue #3)





 
Closed #3 as completed.

**Comment by dahao-1998** [comment]
Dear Mr. Li, I would like to ask why, whether I am training under the TesorFlow framework or the PyTorch framework, when tw=50 and epochs=100, the loss always turns into Nan in the middle. I have tried many solutions online, but none have been resolved. Do you have a solution to this problem?




2595386468
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yaoli90/Conv-CA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月27日(星期一) 晚上10:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yaoli90/Conv-CA] PyTorch model (Issue #3)





 
Closed #3 as completed.

**Comment by yaoli90** [comment]
For the tensorflow version, 
Keras 2.3.1
Tensor flow 1.14.0
should work.

For the PyTorch version, I haven't carefully tested it.

**Comment by dahao-1998** [comment]
Thank you for taking the time to answer me, I will train the model according to your version, and I wish you a brave scientific journey.




2595386468
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yaoli90/Conv-CA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月28日(星期二) 下午2:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yaoli90/Conv-CA] PyTorch model (Issue #3)





 
For the tensorflow version,
 Keras 2.3.1
 Tensor flow 1.14.0
 should work.
 
For the PyTorch version, I haven't carefully tested it.
